src/PIL/PdfImagePlugin.py

```diff
@@ -161,6 +180,11 @@ def _save(im, fp, filename, save_all=False):
 
             if filter == "ASCIIHexDecode":
                 ImageFile._save(im, op, [("hex", (0, 0) + im.size, 0, im.mode)])
+            elif filter == "CCITTFaxDecode":
+                original_strip_size = TiffImagePlugin.STRIP_SIZE
+                TiffImagePlugin.STRIP_SIZE = math.ceil(im.width / 8) * im.height
```

Comment:
In writing this PR, I found that it was necessary to save the image as a single strip. This has actually been pointed out before, in #5740